app/controllers/spree/users_controller.rb

```diff
@@ -101,5 +102,16 @@ def accurate_title
     def user_params
       ::PermittedAttributes::User.new(params).call
     end
+
+    def render_alert_timestamp_error_message
+      render cable_ready: cable_car.inner_html(
+        "#signup-feedback",
+        partial("layouts/alert",
+                locals: {
+                  type: "alert",
+                  message: InvisibleCaptcha.timestamp_error_message
```

Comment:
How is this localised? Or we don't care, assuming only bots will see it?

Reply:
They use `I18n` when available, from the docs 
```
en:
  invisible_captcha:
    sentence_for_humans: "If you are human, ignore this field"
    timestamp_error_message: "Sorry, that was too quick! Please resubmit."
```

So adding the translation in the locales files should work. I have not tried it though.

Reply:
I think the messages  could potentially be shown to users, especially the timestamp_error_message. In that case, they should be translated. In that case, I think we need to copy it to the `en.yml` file, to ensure it will appear in Transifex and get translated.

Also, it's probably worth overriding the default English messages, because bots could easily be trained to look for them. I'm no expert, and am assuming/hoping that most bots are tricked with these methods.
Hmm, how about

```yaml
en:
  invisible_captcha:
    sentence_for_humans: "Please leave empty"
    timestamp_error_message: "Please try again after 5 seconds."
```